utilities/constants.py

```diff
@@ -42,6 +42,14 @@ class Protocols:
     HTTP: str = "http"
     HTTPS: str = "https"
     GRPC: str = "grpc"
+    REST: str = "rest"
+
+
+class HTTPRequest:
+    # Pay attention to the single `'`, needs to be closed after your token
+    # See e.g. tests/model_registry/utils.py#L51
+    AUTH_HEADER: str = "-H 'Authorization: Bearer"
```

Comment:
this will always be used with a token so how about one of:

```
AUTH_HEADER: str = "-H 'Authorization: Bearer {token}'"
AUTH_HEADER.format(token="xx")

```

or
```
from string import Template
 AUTH_HEADER = Template("-H 'Authorization: Bearer $token'")
AUTH_HEADER.substitute(token="xx")

```

Reply:
In every case we need to be aware of the need to handle the token somehow, so I honestly don't see much of a difference. I think your first suggestion is perhaps the cleanest, so I might go with that.